utils/vara-ui/src/components/checkbox/checkbox.module.scss

```diff
@@ -1,36 +1,198 @@
 .label {
   display: flex;
   align-items: center;
+  gap: 8px;
   cursor: pointer;
+  user-select: none;
 
-  &.disabled {
-    pointer-events: none;
-    opacity: 0.3;
+  &:has(.checkbox) {
```

Comment:
Isn't `&:has(.checkbox)` is a redudant here? `.label` always `has` checkbox, am I missing something?